Inside DevOps with Dan Horrocks-Burgess (#1080)
OctopusDeploy/blog: blog/2023-11/inside-devops-dan-horrocks-burgess/index.md

Unlike many stories you hear about DevOps adoption, the business people were really enthusiastic about the changes as they recognized the benefits very early.

**And what does DevOps mean to you?**

*Dan:* Initially, the focus of the DevOps movement was on fostering closer collaboration between development and operations teams. However, we soon recognized the importance of integrating other silos from business and security in the same way.

Over the past 5 years, the trajectory of DevOps has shifted, emphasizing culture, team dynamics, and interactions rather than being solely tool-centric. The human aspect of DevOps is crucial as these factors allow technical practices to impact business outcomes.

Companies who ask for help getting started with DevOps often want to treat it as they do when they engage an external DBA or security testing organization. They want someone to _build DevOps_ for them rather than make changes to their existing structures.

You can't shut the door on a DevOps office, and hope they'll solve this problem. DevOps requires broad changes to the process, technology, team and interactions, and culture. Everyone has a part to play in that.

> **What are some DevOps best practices your organization has implemented?**

Reviewer: @steve-fenton-octopus is this the right question? Feels like Dan's answering a different question and talking generally about organizations that adopt DevOps. Unless he's saying that his organization has fallen short? 
I could be wrong, just want to check. Thanks :)

Author: I think the idea here is...

What are some best practices...

1. Lots of teams have done the basic best practices - source control and automated builds
2. What they aren't doing as much as test and deployment automation (so they should add them)
3. They should also join deployments to the original work items to give the business traceability

So, he's saying source control, build automation (most folks have done this) plus automated tests and deployments (fewer folks have done this) plus infrastructure automation (not implemented to the right level).

There may be a better way to say this.

Reviewer: @steve-fenton-octopus, perhaps the questions should be: What are some DevOps best practices you see people implementing?

Author: Yes! That makes sense as they consult with organizations.

> **What are some DevOps best practices your organization has implemented?**

Reviewer: ```suggestion
**What are some DevOps best practices you see organizations implementing?**
```

When we made dramatic changes to a team in the healthcare industry, it was important to take the business stakeholders along with us. They needed to see that the process served their needs, just as it made the developer experience better.

The easiest part is the tools. It's easy to justify the spend on Octopus as it's easy to calculate the benefits. Your decision-makers will understand articles and white papers on the return on investment of tools like this. It's harder to explain the benefit of the less tangible capabilities of DevOps, like culture and lean product management.

**Can you recommend a good DevOps book?**

*Dan:* Kanban From The Inside by Mike Burrows gives practical insights into the application of Kanban and the reasons behind the practices. It's a useful read for DevOps people as it covers things that happen prior to that all-important commit. It also has useful information for your continuous improvement process.

**What's one thing about you that might surprise us?**

*Dan:* I'm a huge Disney fan and I'm visiting 4 of the 6 global resorts this year: Paris, California, Florida, and Tokyo. I'm hoping to add Shanghai and Hong Kong later on.

:::hint

> If you’d like to be featured in our series, [Inside DevOps](https://octopus.com/blog/tag/Inside%20DevOps), please reach out to [Joanna on LinkedIn](https://www.linkedin.com/in/joannawyganowska/) to set up a time for a quick chat.

Reviewer: @steve-fenton-octopus would you like me to add your name and LinkedIn profile here in addition to Joanna's?

Author: I think we'll leave Joanna's as Inside DevOps is her idea and I don't want to steal the credit. If we change the process we'll have to update all the Inside DevOps footers.
